go.mod

```diff
@@ -160,6 +161,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
+	github.com/golangci/golangci-lint v1.45.2 // indirect
```

Comment:
you accidentally added this as an indirect dependency into go-ipfs rather than updating the test dependency